lib/src/listing/mixins/best.dart

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2017, the Dart Reddit API Wrapper project authors.
```

Comment:
General comment: when adding new files, the copyright year should be the current year.